SpecialOperations2.Altis/sos/actions/functions/fn_parachute.sqf

```diff
@@ -0,0 +1,32 @@
+/*
+ * Author: EnquiringStone [S.O.S. Captain]
+ * Gives the given unit a parachute at a certain height
+ *
+ * Arguments:
+ * 0: vehicle <OBJECT>
+ * 1: unit <OBJECT>
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [_vehicle, _unit] call SOS_fnc_parachute;
+ *
+ */
+
+private ["_vehicle", "_unit", "_height", "_direction", "_parachute"];
+ 
+_vehicle = _this select 0;
+_unit = _this select 1;
+_height = 100;
+_direction = direction _vehicle;
+ 
+moveOut _unit;
+ 
+waitUntil {((getPos _unit) select 2) <= _height};
```

Comment:
getPos is needed here because of this:
if x,y position is over sea, z coordinate is height above the sea.
if x,y position is over land, z coordinate is height above the land.

getPosATL gives me this:
z is the height above terrain level (i.e. for objects in water it's relative to the sea floor).

So when flying above the sea, it will use the sea floor to measure the height. This is not the wanted result